test.py

```diff
@@ -0,0 +1,21 @@
+#while :
```

Comment:
what is this file for - could there be a brief comment to explain what it's for? or walk through it in the readme?

Reply:
Its an old file not longer used.